pkg/util/flags/flags.go

```diff
@@ -81,6 +82,14 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
+func SetFlagsFromCmdline(flags *pflag.FlagSet) {
+	//Add commandline flags to the flagset. We will always write to stderr
```

Comment:
what is the advantage of logging to stderr? I thought the k8s logs picked up stdout

Reply:
glog by default apparently writes to a file and not to stdout. so all k8s logs were getting written to /tmp/ROOK.${logLevel}. Quote from [glog doc](https://godoc.org/github.com/golang/glog)

```
By default, all log statements write to files in a temporary directory. This package provides several flags that modify this behavior. As a result, flag.Parse must be called before any logging is done
```